src/lib/converter/plugins/GitHubPlugin.ts

```diff
@@ -6,6 +6,8 @@ import {Component, ConverterComponent} from '../components';
 import {BasePath} from '../utils/base-path';
 import {Converter} from '../converter';
 import {Context} from '../context';
+import {Option} from '../../utils/component';
+import {ParameterType} from '../../utils/options/declaration';
 
 // This should be removed when @typings/shelljs typings are updated to the shelljs version being used
```

Comment:
Might be a good time to remove manual shelljs typing if possible. Or we could do this in a separate PR.